### DISCUSSION TITLE
MIDI start

### DISCUSSION THREAD

**Discussion by Titouan-dev** [bangcorrupt/hacktribe#261] [Q&A] [answered]
Hi,

I'm doing an external sequenceur with an arduino and I would want to send Play/Pause from my arduino to the hacktribe.

Actually nothing happend and here is my test:
Serial.write(0xB4);
Serial.write(0x63);
Serial.write(0x00);

Serial.write(0xB4);
Serial.write(0x62);
Serial.write(0x05);

Serial.write(0xB4);
Serial.write(0x06);
Serial.write(0x02);

Serial.write(0xB4);
Serial.write(0x26);
Serial.write(0x7f);

if someone can tell me where I'm wrong because I'm going crazy lol

**Comment by bangcorrupt** [comment, marked as answer]
Receiving controls via NRPN is not implemented yet, only sending.  I'll update the wiki to make this clearer.

I think you should be able to use System Real-Time messages to start playback from the Arduino.  

0xfa : Start playback.
0xf8 : Timing clock (keep sending at 24 ppqn).
0xfc : Stop playback.
0xfb : Continue playback (from stopped position without resetting to start).

Set the clock mode to either External USB or External MIDI in Global Settings.
